app/labor/profile_image.rb

```diff
@@ -4,6 +4,7 @@ class ProfileImage
 
   def initialize(resource)
     @resource = resource
+    resource&.profile_image_url
     @image_link = resource.profile_image_url
```

Comment:
this is still going to break, you can just open the console and test it manually by passing nil

Reply:
You right, I just did it